### ISSUE TITLE
phpMyAdmin Does Not Display when Hotlink Protection is Enabled (net::ERR_ABORTED 403)

### ISSUE THREAD

**Issue by jweaver0312** [phpmyadmin/phpmyadmin#18438]
### Describe the bug

This is on a server running CloudLinux and DirectAdmin with phpMyAdmin 5.2.1. When hotlink protection is enabled and blank referrers are not allowed, phpMyAdmin does not render to display anything resulting in error 403 when attempting to load anything for it.

### To Reproduce

1. Enable Hotlink Protection
2. Don't Allow Blank Referrers

### Expected behavior

phpMyAdmin should still render regardless of Hotlink Protection

### Server configuration

- Operating system: CloudLinux
- Web server: LiteSpeed
- Database version: 10.3.28-MariaDB-cll-lve - MariaDB Server
- PHP version: 8.2.5
- phpMyAdmin version: 5.2.1

### Client configuration

- Browser: Chrome 113
- Operating system: macOS Ventura 13.4.1 (a)

**Comment by IOrlandoni** [comment]
The issue you're experiencing is related to [phpMyAdmin@localhost](https://localhost.co/phpmyadmin) not being displayed when Hotlink Protection is enabled. This can result in a net::ERR_ABORTED 403 error. This error occurs because the Hotlink Protection feature prevents the direct access to phpMyAdmin from external sources.

To resolve this issue, you have a few options:

- [x] 1. Whitelist the necessary domains: If you have a list of trusted domains or IP addresses that need access to [phpMyAdmin](https://github.com/IOrlandoni/localhost-phpmyadmin), you can add them to the Hotlink Protection whitelist. This will allow those specific sources to access phpMyAdmin while still blocking others. 
- [x] 2. Disable Hotlink Protection: If you don't require Hotlink Protection for your website, you can disable it entirely. However, keep in mind that this will remove the protection it provides against unauthorized access to your files.
- [x] 3. Adjust Hotlink Protection settings: Some hosting panels or security plugins allow you to customize the Hotlink Protection settings. You might be able to configure it to exclude specific directories or files, such as phpMyAdmin, from being blocked.
- [ ] 4. Consult your hosting provider or refer to the documentation of your website control panel or security plugin for specific instructions on how to modify Hotlink Protection settings.

**Comment by williamdes** [comment]
Hi 

Do you have code examples of what we should change in our code base ?
I am not sure it's on our end to fix things, is it ?

**Comment by jweaver0312** [comment]
Sorry I did get back to this sooner. It looks like phpMyAdmin is showing a bunch of no-referrer policies which can be a direct cause. After playing around with it trying to locate where it would be, I found that in templates/header.twig has a line:

`<meta name="referrer" content="no-referrer">` on line 6 of the file.

With that line removed, everything seems fine.

**Comment by williamdes** [comment]
Hi @jweaver0312 
Thank you for reporting this fix and trying to make a pull-request
@MauricioFauth what do you advise for use to solve this case in QA ?

**Comment by liviuconcioiu** [comment]
@williamdes I don't think this should be removed because it could leak the folder path to outgoing websites, but a new setting maybe in config added for users who want to disable it?
